.circleci/config.yml

```diff
@@ -6,26 +6,24 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8.9
+      - image: node:8.12.0
```

Comment:
Do you want to cut this PR in half and ship the parts that don't require the Node upgrade before you get the Node upgrade?

Reply:
I just downgraded eslint to a version that works with node 8.9.1

Reply:
This is deliberately left as node:8.12.0 as the other image is shipped with yarn 1.3.2. I figured having a forward minor version of node is less destructive than having the wrong version of yarn.